mayavi/__init__.py

```diff
@@ -5,7 +5,7 @@
     Part of the Mayavi project of the Enthought Tool Suite.
 """
 
-__version__ = '4.3.2'
+__version__ = '5.0.0'
```

Comment:
As we upgrade from VTK 5.10.1 to VTK 6, especially using the new pipeline architecture, I thought this qualified for major version upgrade.